.ci/RELEASE_NOTES_GUIDE.md

```diff
@@ -6,162 +6,167 @@ The Magician bot has the ability to copy specifically formatted release notes
 from upstream Magic-Modules PRs to downstream PRs that have CHANGELOGS, namely
 PRs generated in the Terraform providers (GA, beta).
 
-Code lives in magic-modules/downstream_changelog_metadata.py
-
 This guide discusses the style and format of release notes to add
 in PR descriptions so they will be copied downstream and used
-in CHANGELOGs.
+in CHANGELOGs ([GA provider changelog](https://github.com/hashicorp/terraform-provider-google/blob/main/CHANGELOG.md), 
+[beta provider changelog](https://github.com/hashicorp/terraform-provider-google-beta/blob/main/CHANGELOG.md)).
 
 ## Expected Format
 
 The description should have Markdown-formatted code blocks with language
 headings (i.e. text right after the three ticks) like this:
 
 ~~~
-PR Description
-
-...
-
-```release-note:enhancement
-compute: Fixed permadiff for `bar` in `google_compute_foo`
-```
-~~~
-
-You can have multiple code blocks to have multiple release notes per PR, i.e.
-
-~~~
-
-PR Description
-
-...
-
-```release-note:deprecation
-container: Deprecated `region` for `google_container_cluster` - use location instead.
-```
-
-```release-note:enhancement
-container: Added general field `location` to `google_container_cluster`
-```
-~~~
+<PR Description
 
+...>
 
-Do not indent the block and make sure to leave newlines, so you don't confuse
-the Markdown parser. 
-
-To qualify that a change is specific to the beta provider add `(beta)`
-at the end of the release note.\
-This will omit the note from changelogs for the ga release.
-
-~~~
-
-```release-note:enhancement
-compute: added field `multiwriter` to resource `disk` (beta)
+```release-note:enhancement  <-- change type
+compute: added `foo_bar` field to `google_compute_foo` resource <-- change description
 ```
 ~~~
 
+All pull requests need to include release-note block and **change type** (`release-note:REPLACEME` in the PR template) should be replaced with one of the following [headings](#headings), while **change description** can be omitted from some PRs with non-user-facing changes
 
-## Headings
+### Headings
 
-Release notes should be formatted with one of the following headings:
-- `release-note:enhancement`
-- `release-note:bug`
-- `release-note:note`
-- `release-note:new-resource`
-- `release-note:new-datasource`
-- `release-note:deprecation`
-- `release-note:breaking-change`
-- `release-note:none`
+Release notes should be formatted with one of the following headings (See [examples](#examples) below for good release notes for each change type):
```

Comment:
```suggestion
```

I think this sentence can be left out - duplicates information from above. If the examples don't fully fit in the table, it would be great to add a cell in each row with a link to just the example for that type.